client/src/commands.ts

```diff
@@ -141,7 +141,7 @@ export function startLanguageServer(
       extensionContext.testController = undefined;
       extensionContext.statusBar.refresh(extensionContext);
       vscode.commands.executeCommand("setContext", ENABLEMENT_FLAG, false);
-      await client.stop();
+      await client.stop(1e4);
```

Comment:
```suggestion
      const timeoutMs = 10_000;
      await client.stop(timeoutMs);
```

Nitpick: recommend extracting a descriptive variable and not using `e` because it's less familiar.